### DISCUSSION TITLE
Overview of Hugo and Papermod | oliverlabs blog

### DISCUSSION THREAD

**Discussion by giscus** [oliverlabs/oliverlabs.github.io#2] [General]
# Overview of Hugo and Papermod | oliverlabs blog

I found a great post on Hugo and Papermod, explaining the structure and setup of the theme. It can be found here: https://jessewei.dev/blog/2023/papermod/.
Additionally, there’s a decent YouTube video outlining Hugo usage: That’s it for now.

https://oliverlabs.co.uk/posts/hugo/

**Comment by oliverlabs** [comment]
Hi. This is a test comment. Hello, world!
